scanners/zap-extended/templates/zap-extended-scan-type.yaml

```diff
@@ -0,0 +1,95 @@
+---
+{{- if not (empty .Values.zapConfiguration) }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: zap-extended-scantype-config
+  labels:
+    {{- include "zap.labels" . | nindent 4 }}
+data:
+  1-zap-extended-scantype.yaml: {{ .Values.zapConfiguration | toYaml | quote }}
+{{- end }}
+---
+apiVersion: "execution.securecodebox.io/v1"
+kind: ScanType
+metadata:
+  name: zap-extended-scan
+  labels:
+    {{- include "zap.labels" . | nindent 4 }}
+spec:
+  extractResults:
+    type: zap-xml
+    location: "/home/securecodebox/results/zap-results.xml"
+  jobTemplate:
+    spec:
+      {{- if .Values.scannerJob.ttlSecondsAfterFinished }}
+      ttlSecondsAfterFinished: {{ .Values.scannerJob.ttlSecondsAfterFinished }}
+      {{- end }}
+      backoffLimit: {{ .Values.scannerJob.backoffLimit }}
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: zap-extended-scan
+              image: "{{ .Values.scannerJob.image.repository }}:{{ .Values.scannerJob.image.tag | default .Chart.Version }}"
+              imagePullPolicy: {{ .Values.scannerJob.image.pullPolicy }}
+              command:
+                - "python3"
+                - "-m"
+                - "scbzapv2"
+                - "--report-type"
+                - "XML"
+                - "--zap-url"
+                - "localhost:8080"
+                # - "--api-key"
+                # - "ertzukndtzuikbvcfjkmnbvcfghjklmnbvc"
+                - "--config-folder"
+                - "/home/securecodebox/configs/"
+                - "--output-folder"
+                - "/home/securecodebox/results/"
+              resources:
+                {{- toYaml .Values.scannerJob.resources | nindent 16 }}
+              securityContext:
+                {{- toYaml .Values.scannerJob.securityContext | nindent 16 }}
+              env:
+                {{- toYaml .Values.scannerJob.env | nindent 16 }}
+              envFrom:
+                {{- toYaml .Values.scannerJob.envFrom | nindent 16 }}
+              volumeMounts:
+                {{- toYaml .Values.scannerJob.extraVolumeMounts | nindent 16 }}
+              {{- if .Values.scannerJob.extraContainers }}
+              {{- toYaml .Values.scannerJob.extraContainers | nindent 12 }}
+              {{- end }}
+            - name: zap-sidecar
+              image: "{{ .Values.zapContainer.image.repository }}:{{ .Values.zapContainer.image.tag | default .Chart.AppVersion }}"
+              imagePullPolicy: {{ .Values.zapContainer.image.pullPolicy }}
+              command:
+                - "zap.sh"
+                - "-daemon"
+                - "-port"
+                - "8080" 
+                - "-host"
+                - "0.0.0.0"
+                - "-config"
+                - "api.addrs.addr.name=.*" # allow API Access from internal IP's
+                - "-config"
+                - "api.addrs.addr.regex=true"
```

Comment:
This can just be dropped as ZAP defaults to only allow connections from the same host.

Tested this and the scan stills works fine without it, and ZAP will drop any requests from another host.

```suggestion
```